modules/core/src/lib/layer.ts

```diff
@@ -468,8 +468,13 @@ export default abstract class Layer<PropsT = {}> extends Component<PropsT & Requ
 
     const {props, oldProps} = params;
     const neededPickingBuffer =
-      Number.isInteger(oldProps.highlightedObjectIndex) || oldProps.pickable;
-    const needPickingBuffer = Number.isInteger(props.highlightedObjectIndex) || props.pickable;
+      Number.isInteger(oldProps.highlightedObjectIndex) ||
```

Comment:
Again there shouldn't be any knowledge of a particular extension in the core classes. Can we add a method to the LayerExtension interface (e.g. `needsPickingBuffer()`) instead?